misk/src/main/kotlin/misk/cloud/gcp/security/keys/GcpKmsConfig.kt

```diff
@@ -0,0 +1,13 @@
+package misk.cloud.gcp.security.keys
+
+import misk.config.Config
+
+data class GcpKmsConfig(
+        val project_id: String,
+        val key_locations: Map<String, GcpKeyLocation>
+) : Config
+
+data class GcpKeyLocation(val location: String, val key_ring: String, val key_name: String) {
+    val path: String
+        get() = "locations/$location/keyRings/$key_ring/cryptoKeys/$key_name"
```

Comment:
Does this need a getter? Seems like it can just be assigned from the parameters.